client/components/SlideComponentsList/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import * as Components from '@components/Slide/Components';
+import './SlideComponentsList.css';
+
+const SlideComponentsList = ({ components }) => {
+    const style = {
+        height: '150px',
+        overflow: 'hidden'
+    };
+    return (
+        <div style={style}>
+            <svg width="500" height="400">
+                {/* intentionally break out */}
+                <foreignObject
+                    transform="scale(0.5)"
+                    width="100%"
+                    height="100%"
+                >
```

Comment:
This turns the HTML into an SVG everywhere `SlidesListComponents` is used, which covers places outside of the slide previews in the editor (ie in the "Preview" editor tab and on the front end view of a scenario). We'll need to limit this to the slide previews.

Reply:
I totally missed that!